code/modules/events/_event.dm

```diff
@@ -40,6 +41,8 @@
 		return FALSE
 	if(earliest_start >= world.time)
 		return FALSE
+	if(wizardevent != SSevent.wizardmode)
+		return FALSE
```

Comment:
The reverse is also supposed to be true, normal events aren't supposed to run in wizard mode